rust/hyperlane-base/src/types/multisig.rs

```diff
@@ -111,10 +129,15 @@ impl MultisigCheckpointSyncer {
             return Ok(None);
         }
 
-        // Sort in descending order. The n'th index will represent
-        // the highest index for which we (supposedly) have (n+1) signed checkpoints
-        latest_indices.sort_by(|a, b| b.cmp(a));
-        if let Some(&highest_quorum_index) = latest_indices.get(threshold - 1) {
+        // Sort in descending order by index. The n'th index will represent
```

Comment:
nit: I think this comment about the n'th index is no longer correct

Reply:
Don't understand this exactly, I only changed to include the validator alongside the index, how it is incorrect?